### PR TITLE
Content update for individual section

### PR DESCRIPTION
**Pull request by charlotte-hirst**
### What is the context of this PR?
All the content changes, new address questions and amended routing as per slides. I have created a separate card for validation as not needed for translations and looking tricky for me!

### How to review 
Donna has reviewed all content and routes. You can review against slides https://docs.google.com/presentation/d/1BWjHrAfjxgyR69Jwys4BPahm9JF6eUbz2lIkd1se-TQ/edit?usp=sharing

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by jonnyshaw89** [comment]
The content looks sane. i've not run it yet though

File: data/en/lms_2.json [hunk 1]
```diff
@@ -1733,15 +1924,15 @@
...
+                                    "value": "Can <em>{{[answers['first-name'][group_instance], answers['last-name'][group_instance]] | format_household_name}}</em> understand, speak, read or write Welsh?",
                                     "when": [{
                                         "id": "proxy-check-answer",
                                         "condition": "equals",
                                         "value": "proxy"
                                     }]
                                 },
                                 {
-                                    "value": "Can you..."
+                                    "value": "Can you understand, speak, read or write Welsh?"
```
Comment (bitdivision) [inline data/en/lms_2.json:1935]: We could make this conditional on the survey being in welsh. e.g. don't ask it if they're answering in welsh.

**Review by bitdivision — APPROVED** [review]

**Review by jonnyshaw89 — APPROVED** [review]

**Comment by bitdivision** [comment]
@jonnyshaw89 The rebase seems to have broken this. Were there conflicts or anything?

File: data/en/lms_2.json [hunk 2]
```diff
@@ -5709,6 +5999,21 @@
                             ]
                         }]
                     },
+                    {
+                        "id": "feedback-block",
+                        "type": "Question",
+                        "questions": [{
+                            "id": "thank-you-question",
```
Comment (dcdarrell9) [inline data/en/lms_2.json]: Duplicate ID on line 5977,  `"id": "thank-you-question"`

**Pull request merged**